src/ga4gh/vrs/extras/vcf_annotation.py

```diff
@@ -134,6 +134,19 @@ class VCFAnnotator:  # pylint: disable=too-few-public-methods
     VRS_STARTS_FIELD = "VRS_Starts"
     VRS_ENDS_FIELD = "VRS_Ends"
     VRS_STATES_FIELD = "VRS_States"
+    VRS_ERROR_FIELD = "VRS_Error"
+
+    # VCF character escape map
+    VCF_ESCAPE_MAP = {
+        "%": "%25",
+        ":": "%3A",
+        ";": "%3B",
+        "=": "%3D",
+        ",": "%2C",
+        "\r": "%0D",
+        "\n": "%0A",
+        "\t": "%09" 
+    }
```

Comment:
`:`, `=`, and `\r` are not used. Can we remove them? If so, then instead of

```python
                for search in [ "%", ",", ";", "\t", "\n" ]:
                     err_msg = err_msg.replace(search, self.VCF_ESCAPE_MAP[search])
```

we can do

```python
    VCF_ESCAPE_MAP = {
        "%": "%25",
        ";": "%3B",
        ",": "%2C",
        "\n": "%0A",
        "\t": "%09" 
    }
    VCF_ESCAPE_MAP_KEYS = VCF_ESCAPE_MAP.keys()
...
                for search in VCF_ESCAPE_MAP_KEYS:
                     err_msg = err_msg.replace(search, self.VCF_ESCAPE_MAP[search])
```

Reply:
@ehclark do you agree with this feedback? If so, please submit the change if not, please explain why and I'll do the merge.